zerver/migrations/0429_merge_20230211_0759.py

```diff
@@ -0,0 +1,12 @@
+# Generated by Django 4.1.6 on 2023-02-11 07:59
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0426_merge_20230210_1010"),
+        ("zerver", "0428_remove_realm_email_address_visibility"),
+    ]
+
+    operations = []
```

Comment:
You should never create merge migrations. You'll want to delete these and read https://zulip.readthedocs.io/en/latest/subsystems/schema-migrations.html for how to renumber migrations correctly instead.

I know Django's built-in text tells you to merge them, but that's only a thing you want to do if we've already merged both versions! :)

Reply:
Ah alright understood, Django sure made it sound like, the only option I have is to merge migrations lol, anyways fixed the migrations!